apps/aecore/src/aec_keys.erl

```diff
@@ -319,6 +319,12 @@ padding128(Bin) ->
     Pad = 128 - size(Bin),
     <<Bin/binary, 0:(Pad*8)>>.
```

Comment:
please adjust padding here and standarize on leading zeros

Reply:
That would complicate `decrypt_pubkey` slightly.

The padding of the public key is a bit different. In the case of the private key we're padding the actual key (since `crypto:generate_key/2` dropped leading zero bytes), but in the public key case we're padding the block to encrypt. I think it makes sense to do the padding on different sides.

Reply:
ok, thanks